nominations/krzystof-romanowski.md

```diff
@@ -0,0 +1,28 @@
+# Krzysztof Romanowski @romanowski
+
+## Proposer
+
+This candidate has been nominated by Paweł Dolega (@pdolega)
+and seconded by Rory Graves (@rorygraves).
+
+## Qualifications and Experience
+
+- Co-author of Expression Evaluator in Scala IDE's debugger.
+- Involved [work around zinc](https://github.com/sbt/zinc/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Aromanowski+) 
+- Co-organizer of [ScalaSphere](scala.sphere.it) conference that is focused on Scala tooling.
```

Comment:
Notice that this link (and two similar ones below) will be broken. You should add `http://` to the URL.

Reply:
Good point, fixed.
Thanks!